packages/e/e2fsprogs/package.yml

```diff
@@ -1,13 +1,13 @@
 name       : e2fsprogs
-version    : 1.47.2
-release    : 29
+version    : 1.47.3
+release    : 30
 source     :
-    - https://mirrors.edge.kernel.org/pub/linux/kernel/people/tytso/e2fsprogs/v1.47.2/e2fsprogs-1.47.2.tar.xz : 08242e64ca0e8194d9c1caad49762b19209a06318199b63ce74ae4ef2d74e63c
-homepage   : http://e2fsprogs.sourceforge.net
+    - https://mirrors.edge.kernel.org/pub/linux/kernel/people/tytso/e2fsprogs/v1.47.3/e2fsprogs-1.47.3.tar.xz : 857e6ef800feaa2bb4578fbc810214be5d3c88b072ea53c5384733a965737329
+homepage   : https://e2fsprogs.sourceforge.net
 license    :
     - GPL-2.0-only
     - LGPL-2.0-only
-    - BSD
+    - BSD-3-Clause
```

Comment:
This change doesn't seem to be reflected in the pspec file. Was this done after the package was built?